src/test/java/blackjack/domain/card/NumberTest.java

```diff
@@ -0,0 +1,45 @@
+package blackjack.domain.card;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class NumberTest {
+
+    @DisplayName("카드 이름 테스트")
+    @Test
+    void getNameTest() {
```

Comment:
```suggestion
class NumberTest {

    @DisplayName("카드 이름 테스트")
    @ParameterizedTest
    @CsvSource(value = {"ACE,A"})
    void getNameTest(Number number, String name) {
```

`NumberTest` 관련해서 질문 주신 부분에서 원하셨던 방법이 이게 맞는지 모르겠네요. 
그런데 제 생각엔 enum의 필드값을 테스트하는 건 큰 의미는 없다고 생각해요. 이 부분 테스트를 작성해주신 이유가 있을까요?

Reply:
앗.. 최초에 동일한 코드로 작성했을 때엔 작동하지 않았었는데.. 지금은 잘 돌아가는군요..!
블랙잭 게임에선 52장의 카드가 가장 중요한 요소이기 때문에 추후에 누군가 enum 내부를 변경한다면 문제가 있을 것 같아서 모든 enum 이 정확한 값을 갖고 있는지 판단하고자 테스트를 작성했어요! 너무 앞서나간 판단일까요?

Reply:
완벽하게 동일한 정보를 두 군데서 갖고 있는 것 같아서 남긴 코멘트였습니다.
그런 부분을 염두해두신 거라면 수정하실 필요는 없어보이네요 :)